run_testlist.py

```diff
@@ -952,19 +972,27 @@ def _expand_variant(self, env, ppc, variant):
         self._add_partition(env, nodes)
         self._add_timeout(env, timeout)
 
+    def _ec_cell_size_variant(self, env, variant):
+        env['EC_CELL_SIZE'] = str(variant)
+
     def run(self):
         for test in self._testlist:
             if not test['enabled']:
                 continue
 
+            #Set default EC cell size variants
+            if 'ec_cellsize_variants' not in test:
+                test['ec_cellsize_variants'] = ['default']
```

Comment:
Can cellsize only be specified as a number, or does something like "1KB" work as well?
If it's only a number, would it make sense to use `0` as the default? That way we can store a numerical type in the database.

Reply:
That is a good suggestion, The default cell size is 1MB so yes let me change that.